packages/react-native/Libraries/LogBox/UI/AnsiHighlight.js

```diff
@@ -80,32 +82,39 @@ export default function Ansi({
   };
 
   return (
-    <View>
+    <View style={styles.container}>
       {parsedLines.map((items, i) => (
         <View style={styles.line} key={i}>
-          {items.map((bundle, key) => {
-            const textStyle =
-              bundle.fg && COLORS[bundle.fg]
-                ? {
-                    backgroundColor: bundle.bg && COLORS[bundle.bg],
-                    color: bundle.fg && COLORS[bundle.fg],
-                  }
-                : {
-                    backgroundColor: bundle.bg && COLORS[bundle.bg],
-                  };
-            return (
-              <Text style={[style, textStyle]} key={key}>
-                {getText(bundle.content, key)}
-              </Text>
-            );
-          })}
+          <Text>
+            {LRM}
+            {items.map((bundle, key) => {
+              const textStyle =
+                bundle.fg && COLORS[bundle.fg]
+                  ? {
+                      backgroundColor: bundle.bg && COLORS[bundle.bg],
+                      color: bundle.fg && COLORS[bundle.fg],
+                    }
+                  : {
+                      backgroundColor: bundle.bg && COLORS[bundle.bg],
+                    };
+              return (
+                <Text style={[style, textStyle]} key={key}>
+                  {getText(bundle.content, key)}
+                </Text>
+              );
+            })}
+          </Text>
         </View>
       ))}
     </View>
   );
 }
 
 const styles = StyleSheet.create({
+  container: {
+    minWidth: '100%',
```

Comment:
Do we still need to force these to be at least 100% now that the direction is forced to be LTR?

An aside, the offsetting here when content length less than container feels like an Android RTL scrollview bug.

Reply:
https://github.com/facebook/react-native/pull/47280 should fix Android bug here

<img width="647" alt="image" src="https://github.com/user-attachments/assets/e923abf1-dce1-4e6e-be76-1cbeb5430f5a">

Reply:
It’s added to resolve the iOS bug; if you remove this line, it becomes misaligned.
![image](https://github.com/user-attachments/assets/f2f7fd20-3243-4989-851e-583150c20d1d)